.travis.yml

```diff
@@ -7,9 +7,9 @@ cache:
   yarn: true
 
 script:
-  - yarn run lint
-  - yarn run build
-  - yarn run cover
+  - npm run lint
```

Comment:
Yarn seems to be bugged for some reason in Travis
https://travis-ci.org/sourcegraph/javascript-typescript-langserver/builds/210932923

If you know a fix, I have no problem with changing it back, but this shouldn't make any difference